src/components/Image/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// TODO: get from site branding
+import settings from '../../data/configuration/constants';
+
+import './Image.scss';
+
+
+const Image = props => (
+  <div id="image-border" className="w-100">
+    { (props.title.trim() !== '') ? <div id="image-title">{props.title} </div> : '' }
```

Comment:
This is checking that the title is not an empty string or just a string of whitespaces.  It will only display that div if ```props.title``` has some actual content.  I'll add a comment so that is clearer.

Reply:
I guess my question was more asking whether it was better to have an `image-title` be absent, rather than blank. I think it'd make sense to tell that the title is empty as long as it doesn't mess with design. Kind of like how it's better to have a blank alt-text than not have one. 

I'm fine with it as is though, just wanted to ask what you thought.